vaadinfrontend/src/main/java/life/qbic/datamanager/views/support/experiment/ExperimentItemCollection.java

```diff
@@ -66,9 +62,8 @@ public static ExperimentItemCollection create(String labelCreationCard) {
    */
   public void addExperimentItem(ExperimentItem item) {
     items.add(item);
-    add(item);
-    addComponentAsLast(createExperiment);
-    subscribeToClickEvent(item);
+    addComponentAsFirst(item);
+    addSelectionEventListener(item);
```

Comment:
why did you change the behaviour here? from addin it in the end of the list to adding it on top? (not part of this PR I would say)

Reply:
was this not already the case?